man/deepsea-stage-show.1

```diff
@@ -0,0 +1,69 @@
+.TH deepsea-stage-show 1 "DeepSea CLI man page"
+.SH NAME
+deepsea-stage-show - Shows DeepSea stage execution steps
+
+.SH SYNOPSIS
+deepsea stage show [--help] [--hide-state-steps] [--only-visible-steps]
+                   [--clear-cache] [--no-cache]
+                   <stage_name>
+
+.SH DESCRIPTION
+Shows a DeepSea stage steps ordered by execution order.
+
+This command can be used to understand what are the steps involved in each
+stage. For instance, it shows the total number of steps; the type of each step,
+either
+.I Runner,
+.I State,
+or
+.I Module;
+in which minion each step will execute; and other relevant information.
+
+.SH OPTIONS
+.B --help
+.RS
+Shows the help message.
+
+.RE
+.B --hide-state-steps
+.RS
+Disables the parsing of Salt state intermediate steps.
+
+.RE
+.B --only-visible-steps
+.RS
+Only show state operations that declared
```

Comment:
is there a difference between "operation" and "step" of a stage? If not, let's use "step" to keep the terminology consistent.

"Only show steps that have `fire_event=True` in their Salt configuration." ?